flatgfa/src/flatgfa.rs

```diff
@@ -101,6 +101,12 @@ pub struct Path {
     pub overlaps: Span<Span<AlignOp>>,
 }
 
+impl Path {
+    pub fn step_count(&self) -> usize {
+        self.steps.end.index() - self.steps.start.index()
```

Comment:
I think `self.steps.len()` does this. (In fact, maybe we don't need `path.step_count()` if `path.steps.len()` is one character shorter? :smiley:)